.buildkite/pipeline.yml

```diff
@@ -85,6 +87,9 @@ steps:
 
           .buildkite/commands/install-node-dependencies.sh
 
+          # Ensure DEV_BUILD is available to npm commands
+          export DEV_BUILD=true
```

Comment:
I'm not 100% sure we should add DEV_BUILD to Mac and Windows builds. Are we adding it to avoid Sentry to upload the map?

We should double check if adding DEV_BUILD=true to Mac and Windows build won't affect to Sentry initialization for real users.

Reply:
> I'm not 100% sure we should add DEV_BUILD to Mac and Windows builds. Are we adding it to avoid Sentry to upload the map?

Yes, according to [Sentry's documentation](https://docs.sentry.io/cli/releases/#commit-integration):

> Releases can also be auto created by different systems. For instance upon uploading a source map a release is automatically created. Likewise releases are created by some clients when an event for a release comes in.